samples/powerstore_v230_k8s_122.yaml

```diff
@@ -0,0 +1,121 @@
+apiVersion: storage.dell.com/v1
+kind: CSIPowerStore
+metadata:
+  name: csi-powerstore
```

Comment:
Please change the name test-powerstore to be consistent with the n-2 samples that are there in codebase